tests/bdd/scale/test_NAS_T1062.py

```diff
@@ -111,14 +114,21 @@ def click_the_start_automatically_ssh_checkbox_and_enable_the_ssh_service(driver
     value_exist = attribute_value_exist(driver, '//mat-slide-toggle[@ix-auto="slider__state__SSH"]', 'class', 'mat-checked')
     if not value_exist:
         driver.find_element_by_xpath('//div[@ix-auto="overlay__stateSSH"]').click()
-    time.sleep(1)
+    time.sleep(4)
```

Comment:
I think I was not clear. this time.sleep(4) can be removed.

plus the section to go to the dashboard it is an unnecessary step.